WorkloadManagementSystem/Service/JobMonitoringHandler.py

```diff
@@ -31,16 +33,28 @@
 def initializeJobMonitoringHandler(serviceInfo):
 
   global gJobDB, gJobLoggingDB, gTaskQueueDB
+
   gJobDB = JobDB()
   gJobLoggingDB = JobLoggingDB()
   gTaskQueueDB = TaskQueueDB()
+
   return S_OK()
 
 
 class JobMonitoringHandler(RequestHandler):
 
   def initialize(self):
 
+    global gJobElasticDB, gJobDB
+
+    gESFlag = self.srv_getCSOption('useES', False)
+    if gESFlag:
+      gJobElasticDB = JobElasticDB()
+
+    gMySQLFlag = self.srv_getCSOption('useMySQL', True)
+    if not gMySQLFlag:
+      gJobDB = False
+
```

Comment:
I just ask one more thing: in the docstring at the beginning of the module, please add a 1-or-2 lines documentation on these 2 flags.